pkg/cimfs/mount_cim.go

```diff
@@ -0,0 +1,78 @@
+//go:build windows
+// +build windows
+
+package cimfs
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
+	"github.com/Microsoft/hcsshim/internal/winapi"
+	"github.com/pkg/errors"
+)
+
+// This package provides simple go wrappers on top of the win32 CIMFS mount APIs. The mounting/unmount of cim
+// layers is done by the cim mount functions in internal/wclayer/cim package.
```

Comment:
This comment is out of date right?

Reply:
No. The next PR adds a package at `internal/wclayer/cim` that brings in the cim layer mount/unmount functionality.